src/Core/src/Primitives/FilterMode.cs

```diff
@@ -2,8 +2,8 @@ namespace Microsoft.Maui
 {
 	public enum FilterMode
 	{
-		SrcIn,
-		Multiply,
-		SrcAtop
+		SrcIn = 0,
+		Multiply = 1,
+		SrcAtop = 2,
```

Comment:
These are *the exact same* as before, but because we switch on these values in Java I feel better if we explicitly list the values.